node_modules/col-assets/tests/util.js

```diff
@@ -100,6 +110,16 @@ var assertAsset = module.exports.assertAsset = function(asset, expectedAsset, ex
       assert.strictEqual(asset.views, expectedAsset.views);
     }
 
+    // Ensure that the expected categories are present
+    if (expectedCategories) {
```

Comment:
there are 2 `if (expectedCategories) {` blocks in here, which tripped me up. Can those be merged?